cmd/zoekt-sourcegraph-indexserver/queue.go

```diff
@@ -129,13 +149,21 @@ func (q *Queue) Bump(ids []uint32) []uint32 {
 		if !ok {
 			missing = append(missing, id)
 		} else if item.heapIdx < 0 {
-			q.seq++
-			item.seq = q.seq
-			item.dateAddedToQueue = time.Now()
-
-			heap.Push(&q.pq, item)
-			metricQueueLen.Set(float64(len(q.pq)))
-			metricQueueCap.Set(float64(len(q.items)))
+			if item.backoffUntil.After(time.Now()) {
+				q.logger.Debug("Backoff enqueue of repository",
+					sglog.String("repo", item.opts.Name),
+					sglog.Uint32("id", id),
+					sglog.Time("backoff_until", item.backoffUntil),
+				)
```

Comment:
we call bump pretty regularly (every minute?) so these logs might be quite noisy.

Reply:
Yeah that's a good point, even for debug it's noisy. I've removed this. The backoff.Fail() will still log when we are going to apply a backoff until timestamp in the future, and log that timestamp, so we will still have something in the logs to look for